UI-Session-6-Assignment/index.html

```diff
@@ -0,0 +1,881 @@
+<!DOCTYPE html>

```

Comment:
1. Check the alignment of Advanced UI and Component UI.
2. There is no scroll bar at Annex session.
3. Check the alignment of search button.

Reply:
Done